.travis.yml

```diff
@@ -4,7 +4,6 @@ rust:
     - 1.15.0
     - stable
     - beta
-    - nightly
```

Comment:
Oh. When you said you would remove the nightly feature I thought you meant the feature only, not actually stop building on nightly. What stops us from building on nightly? We don't need to use the nighly magic compiler plugin stuff, but I still think we should build on nightly to make sure it builds. Or is there any reason we can't build on nightly?

Reply:
I actually removed this by mistake. I considered adding it back, but I liked the idea of faster builds. I'll add it back.

Reply:
We are going to get faster builds anyway I assume, Since you simplified the matrix of possible environments. Yeah, I don't really like the idea of not testing against nightly, basically everyone test against stable, beta and nightly for good reasons.